TEAM.md

```diff
@@ -0,0 +1,56 @@
+# Team membership
+
+Current members are:
+
+- `@alexbit-codemod`
+- `@AugustinMauroy`
+- `@avivkeller`
+- `@bmuenzenmeyer`
+- `@JakobJingleheimer` (lead)
+- `@ljharb`
+
+## Rights
+
+Any team member may block a PR or decision. A block may be overturned by team vote; votes require a quorum of ⅔ of active members and pass by >½ simple majority. In accordance with Node.js policy, a team's vote may be overturned by the [TSC](https://github.com/nodejs/TSC).
```

Comment:
```suggestion
Any team member can block a pull request or decision. A block can be overturned by a team vote, which requires a quorum of two-thirds of active members and passes with a simple majority (>50%). Per Node.js policy, the [TSC](https://github.com/nodejs/TSC) may override a team’s vote.
```

Reply:
Nit: `Per Node.js policy` → `Pursuant to Node.js policy`